### ISSUE TITLE
17.4.0 Wheel build fails due to LICENSE file missing

### ISSUE THREAD

**Issue by cooperlees** [PyCQA/flake8-bugbear#32]
I'm unable to build a wheel due to LICENSE file missing. Will investigate and fix.

```
Building wheels for collected packages: flake8-bugbear, flake8, attrs, pyflakes, pycodestyle, mccabe, setuptools
  Running setup.py bdist_wheel for flake8-bugbear: started
  Destination directory: /tmp/tmpcxf7s8htpip-wheel-
  Running command /tmp/pyfi/pyfi_venv/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-h5tf698v/flake8-bugbear/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpcxf7s8htpip-wheel- --plat-name=gcc_5_glibc_2_23
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib
  copying bugbear.py -> build/lib
  installing to build/bdist.linux-x86_64/wheel
  running install
  running install_lib
  creating build/bdist.linux-x86_64
  creating build/bdist.linux-x86_64/wheel
  copying build/lib/bugbear.py -> build/bdist.linux-x86_64/wheel
  running install_egg_info
  running egg_info
  writing flake8_bugbear.egg-info/PKG-INFO
  writing dependency_links to flake8_bugbear.egg-info/dependency_links.txt
  writing entry points to flake8_bugbear.egg-info/entry_points.txt
  writing requirements to flake8_bugbear.egg-info/requires.txt
  writing top-level names to flake8_bugbear.egg-info/top_level.txt
  reading manifest file 'flake8_bugbear.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  warning: no files found matching '*.txt' under directory 'tests'
  writing manifest file 'flake8_bugbear.egg-info/SOURCES.txt'
  Copying flake8_bugbear.egg-info to build/bdist.linux-x86_64/wheel/flake8_bugbear-17.12.0-py3.6.egg-info
  running install_scripts
  error: [Errno 2] No such file or directory: 'LICENSE'
  Running setup.py bdist_wheel for flake8-bugbear: finished with status 'error'
```

**Comment by ambv** [comment]
Fixed by #30. Will release 18.2.0 now with it.

**Comment by ambv** [comment]
18.2.0 released.
